src/features/projects/components/maps/StyleToggle.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { ReactElement } from 'react';
+import { ProjectPropsContext } from '../../../common/Layout/ProjectPropsContext';
+import styles from '../../styles/ProjectsMap.module.scss';
+
+interface Props {}
+
+export default function StyleToggle({}: Props): ReactElement {
+  const { satellite, setSatellite } = React.useContext(ProjectPropsContext);
+  return (
+    <div className={styles.layerSwitcher}>
+      <div
+        onClick={() => setSatellite(false)}
+        className={`${styles.layerOption} ${satellite ? '' : styles.active}`}
+      >
+        Map
+      </div>
+      <div
+        onClick={() => setSatellite(true)}
+        className={`${styles.layerOption} ${satellite ? styles.active : ''}`}
+      >
+        Satellite
```

Comment:
Please use some translation resource here.

Reply:
this file is no longer being used. deleted it